### DISCUSSION TITLE
ENV NVIDIA_REQUIRE_CUDA Question

### DISCUSSION THREAD

**Discussion by Artekus** [blakeblackshear/frigate#1887] [Ask A Question]
Does the last line as follows in the Dockerfile.amd64nvidia cause it to be limited to tesla and if so is there away to remove the brand requirement?
`ENV NVIDIA_REQUIRE_CUDA "cuda>=11.1 brand=tesla,driver>=418,driver<419 brand=tesla,driver>=440,driver<441 brand=tesla,driver>=450,driver<451"`

**Comment by wills106** [comment]
I have used the Nvidia branch in the past with a GTX1650 without issues.

Maybe it's just how the CUDA calls it's self, as it perhaps started off on Tesla cards?
